model.py

```diff
@@ -1,150 +1,261 @@
+import sqlite3
+import json
+import random
+
 class Model:
-	def __init__(self):
-		'''
-		Constructor 
-		'''
-		pass
-
-	def set_user_info(self, nutrition_standard,
-                           height,
-                           weight,
-                           age,
-                           meal_list):
-        """
-        Store user info from surveys to database
-        @param
-        nutrition_standard: String
-        height: Int
-        weight: Int
-        age: Int
-        meal_list: List<String>
-
-        Database: UserInfo
-        @column
-        nutrition_standard: String
-        height: Int
-        weight: Int
-        age: Int
-        meal_list: List<String>
-        """
+    connection  = sqlite3.connect('mealplan.db')
+    cursor = connection.cursor() 
+    
+    #the meal_list has to be string because sqlite cannot store list datatype
+    #then using the json library to deserialize the meal list into string type
+
+    command_createUserInfo = '''
+        CREATE TABLE IF NOT EXISTS 
+        UserInfo(ID INTEGER PRIMARY KEY AUTOINCREMENT, nutrition_standard TEXT, height INTEGER, weight INTEGER, age INTEGER, meal_list TEXT)
+    ''' 
+
+    command_createUserDefinedMeal = ''' 
+        CREATE TABLE IF NOT EXISTS
+        UserDefinedMeal(meal_name TEXT, start_time TEXT, end_time TEXT, 
+                        set_of_dishes TEXT, nutritious_retriction TEXT, 
+                        regular BOOLEAN, flexible BOOLEAN)
+    ''' 
+
+    command_createRecipe = '''
+        CREATE TABLE IF NOT EXISTS
+        Recipe(recipe_name TEXT, serving_size INTEGER, cooking_time INTEGER, tag TEXT,
+                ingredient_name TEXT, amount FLOAT, nutrition_name TEXT,
+                energy FLOAT, steps_taken TEXT)
+    '''
 
+    cursor.execute(command_createUserInfo)
+    cursor.execute(command_createUserDefinedMeal) 
+    cursor.execute(command_createRecipe)
+
+    connection.commit()
+
+    def __init__(self):
         pass
 
-    def get_user_info(self):
+    def set_user_info(UserInfo):
```

Comment:
change `UserInfo` to `user_info`, because it is just a variable

Reply:
done